notebooks/Part 1 - Data Exploration & Visualization.ipynb

```diff
@@ -0,0 +1 @@
zamftm',\n 'asset_id': 'cc58d6de-60da-4e5c-93eb-937ca9992461'}\n```\n\nand\n\n```\n{'file_name': 'groups_of_themes.csv',\n 'message': 'File saved to project storage.',\n 'bucket_name': 'ibmdebaterthematicclusteringofsen-donotdelete-pr-wsojkqqhzamftm',\n 'asset_id': '2f8dabad-709a-4cce-8454-80e59c7ca459'}\n```\n\n**Note**: In order for this step to work, your project token (see the first cell of this notebook) must have `Editor` role. By default this will overwrite any existing file."}, {"metadata": {"scrolled": true}, "cell_type": "code", "source": "project.save_data(\"themes.csv\", df.to_csv(index=False, float_format='%g'), overwrite=True)", "execution_count": 20, "outputs": [{"output_type": "execute_result", "execution_count": 20, "data": {"text/plain": "{'file_name': 'themes.csv',\n 'message': 'File saved to project storage.',\n 'bucket_name': 'ibmdebaterthematicclusteringofsen-donotdelete-pr-wsojkqqhzamftm',\n 'asset_id': '55b504c4-d01f-477e-909b-ac5326d4b4e3'}"}, "metadata": {}}]}, {"metadata": {}, "cell_type": "code", "source": "project.save_data(\"groups_of_themes.csv\", groups_of_themes.to_csv(index=False), overwrite=True)", "execution_count": 21, "outputs": [{"output_type": "execute_result", "execution_count": 21, "data": {"text/plain": "{'file_name': 'groups_of_themes.csv',\n 'message': 'File saved to project storage.',\n 'bucket_name': 'ibmdebaterthematicclusteringofsen-donotdelete-pr-wsojkqqhzamftm',\n 'asset_id': '7a911454-0b9c-49f1-af76-6584d5f07f2f'}"}, "metadata": {}}]}, {"metadata": {}, "cell_type": "markdown", "source": "\n#### Next steps\n\n- Close this notebook.\n- Open the `Part 2 - Model Development` notebook to explore the cleaned dataset.\n\n\n\n<a id=\"authors\"></a> \n### Authors\nThis notebook was created by the [Center for Open-Source Data & AI Technologies](http://codait.org).\n\nCopyright \u00a9 2020 IBM. This notebook and its source code are released under the terms of the MIT License."}, {"metadata": {}, "cell_type": "markdown", "source": "<div style=\"background:#F5F7FA; height:110px; padding: 2em; font-size:14px;\">\n<span style=\"font-size:18px;color:#152935;\">Love this notebook? </span>\n<span style=\"font-size:15px;color:#152935;float:right;margin-right:40px;\">Don't have an account yet?</span><br>\n<span style=\"color:#5A6872;\">Share it with your colleagues and help them discover the power of Watson Studio!</span>\n<span style=\"border: 1px solid #3d70b2;padding:8px;float:right;margin-right:40px; color:#3d70b2;\"><a href=\"https://ibm.co/wsnotebooks\" target=\"_blank\" style=\"color: #3d70b2;text-decoration: none;\">Sign Up</a></span><br>\n</div>"}], "metadata": {"kernelspec": {"name": "python3", "display_name": "Python 3.7", "language": "python"}, "language_info": {"name": "python", "version": "3.7.9", "mimetype": "text/x-python", "codemirror_mode": {"name": "ipython", "version": 3}, "pygments_lexer": "ipython3", "nbconvert_exporter": "python", "file_extension": ".py"}}, "nbformat": 4, "nbformat_minor": 4}
```

Comment:
<p>Let's leave the bucket_name as </p><pre class="ql-syntax" spellcheck="false">'bucket_name': 'ibmdebaterthematicclusteringofsen.....'
</pre><p>Also, asset_id as</p><pre class="ql-syntax" spellcheck="false">'asset_id': '...'
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-thematic-clustering-project/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>